Gruntfile.js

```diff
@@ -188,6 +177,21 @@ module.exports = (grunt) => {
     }
   });
 
+  grunt.registerTask('checkPrettier', 'Verify that all files are correctly formatted.', function() {
+    const done = this.async();
+    childProcess.exec('prettier -l . bin/*', (err, stdout, stderr) => {
```

Comment:
I would prefer using the [prettier check API](https://prettier.io/docs/en/api.html#prettierchecksource--options) instead of childProcess.

Also, if we are using a command anyway, I do not see much value by adding it to Grunt.

Reply:
Yeah, I just tried to run `grunt` but it didn't work because I don't have `prettier` installed globally. Only `npm run build` works now.

Reply:
While I agree that that is the cleaner way of going about it, it is also a lot more work and I just want to get code formatted :)

On top of that, it's not clear that Grunt will stay in use. Personally, I havent used Grunt since 2014, and instead I rely on Webpack and nps for my entire workflow. Grunt is just so much more complex.

And even if Grunt will stay in use, eslint is imho better than jshint, and it has prettier support for checking and fixing.

I added `npx` to the grunt command so that it will use the installed prettier.